client/README.md

```diff
@@ -26,10 +26,6 @@ To see it in action, check out the [demo server](https://demo-open.msupply.org/)
 
 `yarn start API_HOST='http://localhost:8001'` (using custom API url, see [config.ts for more info](./packages/config/src/config.ts))
 
-- If there are feature flags in use, they are set as environment variables. For example:
-
-`yarn start FEATURE_INVENTORY_ADJUSTMENTS=true`
```

Comment:
have replaced the old implementation, as you'll see below

Reply:
Ah great shout, thanks! I think as feature flags were not widely used, I'm okay with not commenting on the old pattern, but have added a note to the frontend readme about how they work now!